core/src/main/java/org/mskcc/cbio/portal/scripts/ImportCopyNumberSegmentData.java

```diff
@@ -144,7 +144,14 @@ public void run() {
     
     private void importFractionGenomeAltered(CancerStudy cancerStudy) throws DaoException {
 
-        DaoCopyNumberSegment.calculateFractionGenomeAltered(cancerStudy.getInternalId());
+        ClinicalAttribute attr = new ClinicalAttribute("FRACTION_GENOME_ALTERED", "Fraction Genome Altered", "Fraction Genome Altered", "NUMBER",
+            false, "1", cancerStudy.getInternalId());
```

Comment:
@zhx828 what's the priority for "FRACTION_GENOME_ALTERED" in the study view?

Reply:
https://github.com/cBioPortal/cbioportal/blob/master/portal/src/main/webapp/js/src/dashboard/resources/configs.json#L29 seems like it's 20